File: src/Api/Model/DB/QualificacaoResponsavel.cs
using Realms;
using System;
using System.Collections.Generic;
using System.Text;

namespace CriadorBaseDados.Model.DB
{
    public class QualificacaoResponsavel : RealmObject
    {
        [PrimaryKey]
        public int Cod { get; set; }
        public string Descricao { get; set; }
    }
}

File: src/Mobile/b2b/lib/model/core/qualificacao_responsavel.dart
class QualificacaoResponsavel {
  final int cod;
  final String descricao;

  QualificacaoResponsavel(this.cod, this.descricao);

  QualificacaoResponsavel.fromJson(Map<String, dynamic> json)
      : cod = json['cod'],
        descricao = json['descricao'];

  Map<String, dynamic> toMap() => <String, dynamic>{
        'cod': cod,
        'descricao': descricao,
      };
}
